crates/swc_stylis/tests/prefixer.rs

```diff
@@ -458,15 +458,15 @@ fn error_recovery_1() {
             animation: slide 3s ease infinite;
         ",
         "__styled-jsx-placeholder__1
-            animation: slide 3s ease infinite;",
+            animation: slide 3s ease infinite",
     );
 
     t(
         "animation: slide 3s ease infinite;
             __styled-jsx-placeholder__1
         ",
         "-webkit-animation:slide 3s ease infinite;animation:slide 3s ease \
-         infinite;__styled-jsx-placeholder__1\n        ;",
+         infinite;__styled-jsx-placeholder__1\n        ",
```

Comment:
Want to verify it will not break nothing, don't know context of `// This behavior is wrong, but it's what `stylis@3` does.`, should we print `;` in this case or not, in CSS only latest declaration can omit `;`

Currently we print `;` in all valid cases where it is valid, but we will improve it in future - for `minify: true` we will omit `;` at the end and decrease size of minified CSS

Reply:
Yeah, we should emit `;` if it's not last because it will be replaced by style property like `color: red`.

To be exact, `__styled-jsx-placeholder__1` becomes `color: red`